### DISCUSSION TITLE
[Scam Alert] Smart World

### DISCUSSION THREAD

**Discussion by FeelsDejavu** [Webamoozcom/warning-list#134] [Investigation بررسی]
**نام پروژه:** Smartworld.io

**نماد پروژه:** STTS

**اتفاقی که افتاده است:**
در گذشته پروژه های پانزی مبنی بر اسمارت کانتراکت ها که روی شبکه ترون معروف بود همانند بانک آف ترون سرمایه زیادی از افراد رو کلاهبرداری کردند. این نوع پروژه ها با تشویق سرمایه گذاری و جذب افراد برای سود های نجومی ماهانه شما رو ترقیب به سرمایه گذاری میکنند
پیشنهاد سرمایه گذاری این صورت که  با ۱۰۰۰ دلار یا بیشتر در مدت ۱۵ ماه پول شما دوبرابر میشه و روزانه میتوانید سود رو نقد کنید . سود تضمینی ۱۴ درصد به صورت ماهانه میده بدون زیرمجموعه گرفتن و اگر زیرمجموعه گیری انجام دادیم سود بسیار زیادی میده. 
![IMG_20220514_174541_646.png](https://user-images.githubusercontent.com/105551521/168439062-f8919875-9ad3-458c-8314-b2a95a6c8d7e.png)

اسم توکن stts هست و اسم سیستم smartworld.app
در شیراز بسیار گسترش پیدا کرده و مرتب جلسات گسترده میزارند و تبلیغ کننده هاش میگن چون ۱۱ ماهه کار میکنه و با bnb نقد میشه و قرارداد هوشمند  و وایت پیپر داره بهیچ عنوان کلاهبرداری نیست.
این پروژه ایرانی نیست ولی گروهی داخل ایران در حال پروموت این پلتفرم هستند و افراد زیادی رو زیر مجموعه گیری میکنند

بعد از بررسی به این نتیجه رسیدیم که توکن بالای ۹۰ درصد در اختیار خود پلتفرم هست:
https://bscscan.com/token/0x88469567a9e6b2dae2d8ea7d8c77872d9a0d43ec?a=0x0178ba5b863f343965e5ffafa988f86e69d32f2e
![Screenshot_20220514-173515_Chrome.jpg](https://user-images.githubusercontent.com/105551521/168438073-125951e1-679f-487c-ba00-27f5aa29e50e.jpg)
ولت اولیه ولت خود پلتفرم هست که تراکنش مستقیم با ادرس ولت خود سایت داره.
ولت سایت:
![IMG_20220514_174543_117.png](https://user-images.githubusercontent.com/105551521/168439100-35d04942-475c-40bc-9216-458eba396a4c.png)

ولت اولی که بالای ۷۸ درصد از ساپلای توکن رو دارد
![IMG_20220514_174544_259.png](https://user-images.githubusercontent.com/105551521/168439134-9c54cf3d-b29a-476c-a729-7f3b51596367.png)
 در تراکنش دوم ۲۰ bnb به ولت سایت ارسال شده و نشون دهنده مالکیت این ولت به سایت هست.
در بررسی انتهایی بالای ۱۲۰ bnb از هات ولت بایننس به ولت های این افراد ارسال شده و چندجا نقد شده و خارج شده.
45bnb استخر توکن هست که متغیر نیز هست
**قوانینی که نقض شده است:** سود غیر منطقی سالانه و ماهانه بدون پشتیبان و زیر مجموعه گیری پانزی.

**نوع هشدار:** کلاهبرداری

**نوع کلاهبرداری:** پانزی اسکم

**وضعیت کلاهبرداری:** در حال اجرا

**نام کلاهبرداران:** مشخص نیست

**آدرس وب سایت(ها):** smartworld.app

**منابع:** bscscan.com

**جمع بندی: کلابرداری لطفا اطلاع رسانی بشه و سرمایه گذاری نکنید

**Comment by micro73** [comment]
ادرس سایت smartworld.app
ادرس توییتر @smart_world_app
ادرس گیت هاب https://github.com/smartworld2030
-	ایمیل داده شده در گیت هاب : [smartworldapplication@gmail.com](mailto:smartworldapplication@gmail.com)
-	ایمیل داده شده در سایت پروژه info@smartworld.app

توکن بر روی شبکه بایننس اسمارت چین بانام  Smart World Token - Stock (STTS)  و با ادرس کانترکت 
0x88469567A9e6b2daE2d8ea7D8C77872d9A0d43EC پیاده سازی شده است.


تعداد کل توکن ها : 100,000,000  STTS

هنوز در کوین مارکت کپ لیست نشده است.

تعداد هولدر 1,731 addresses

حدود 79 درصد توکن ها در یک ولت می باشد که این به شدت خطرناک است.

در سایت پروژه در ابتدای ورود درخواست اتصال به کیف پول میدهد که خطرناک است همچنین طراحی سایت به عنوان پروژه خوب در خیلی کیفیت پایینی دارد.

در سایت پروژه ادرس کانال تلگرام هم وجود دارد که با کلیک بر روی ان هیچ کانالی باز نمیشود.

دارای دو شبکه اجتماعی اینستاگرام و توییتر است که اینستاگرام دارای 94 فالوور است و توییتر دارای 61 فالوو است به گفته خودشون در توییتر علاوه بر شبکه باییننس روی شبکه icpهم پیاده سازی شده است ولی در سایت ادرس توکن برای این شبکه نیست و هنگامی که در سایت https://www.icpexplorer.org/ اسم توکن را سرچ میکنید وجودی ندارد.

سازندگان پروژه به هیچ عنوان مشخص نیستند چه اشخاصی هستند و هویتشان پنهان است.

وایت پیپر پروژه : دارای غلط املایی است .

صحبت های یک فرد که دعوت شده به سرمایه گذاری : 
پیشنهاد سرمایه گذاری به من شده به این صورت که  با ۱۰۰۰ دلار یا بیشتر در مدت ۱۵ ماه پولمون دوبرابر میشه و روزانه میتونیم سود رو نقد کنیم . سود تضمینی ۱۴ درصد به صورت ماهانه میده بدون زیرمجموعه گرفتن و اگر زیرمجموعه گیری انجام دادیم سود بسیار زیادی میده. 
اسم توکن stts هست و اسم سیستم smartworld.app
در شیراز بسیار گسترش پیدا کرده و مرتب جلسات گسترده میزارن و تبلیغ کننده هاش میگن چون ۱۱ ماهه کار میکنه و با بایننس نقد میشه و قرارداد هوشمند  و وایت پیپر داره بهیچ عنوان کلاهبرداری نیست.

**Comment by FeelsDejavu** [comment]
این پروژه به لیست هشدار  @Blockunity هم اضافه میشود

**Reply by micro73** [reply]
خوب ميكني 
اين مورد رو فراموش كردم اضافه كنم اونجا
